### PR TITLE
Publish pkg to nixos

### PR DESCRIPTION
**Pull request by tonyshkurenko**
Publish pkg to nixpkgs

### CODE REVIEW

**Comment by tonyshkurenko** [comment]
Not latest version (1.0.21), because we haven't tested headless on nixos

File: pkgs/applications/networking/twingate/default.nix [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+, stdenv
+, pkgs
+}:
+
+stdenv.mkDerivation rec {
+  pname = "twingate";
+  version = "1.0.12";
+  
+  src = fetchurl {
+    url = "gs://twingate-prod-binaries/client/linux/DEB/1.0.12/twingate-amd64.deb";
```
Comment (sagie-shamay) [inline pkgs/applications/networking/twingate/default.nix:18]: We should use the public endpoint and not the GCS endpoint. How does it even work? It is a `gs` scheme and not `https` 🤔

**Review by alexmensch — APPROVED** [review]

File: pkgs/applications/networking/twingate/default.nix [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    mv usr/share $out/share
+  '';
+
+  meta = with lib; {
+    description = "Twingate Remote Access Client";
+    homepage = "https://twingate.com";
+    license = licenses.unfree;
+    maintainers = [ {
+        name = "Anton Shkurenko";
+        email = "anton@twingate.com";
```
Comment (tonyshkurenko) [inline pkgs/applications/networking/twingate/default.nix:50]: We probably better list some of our support emails

**Comment by tonyshkurenko** [comment]
I've tested it yesterday, but to test it (if you want), clone this PR and follow those steps:
https://www.notion.so/twingate/NixOS-Installation-Steps-f8157c0e0be94623957a9346b288e206#8865b92f44894c9693b237d093c24c23

To clone PR (without downloading tons of files): 

1) `git init`
2) `git remote add origin https://github.com/Twingate/nixpkgs/`
3) `git pull origin pull/1/head:branch_from_pr_name --depth=1`

File: pkgs/applications/networking/twingate/default.nix (hunk 2 above)
Comment (sagie-shamay) [reply pkgs/applications/networking/twingate/default.nix:50]: @liorr  @alexmensch Do we want to use `support@twingate.com`? Create another email address for that?

Comment (tonyshkurenko) [reply pkgs/applications/networking/twingate/default.nix:50]: https://github.com/Twingate/client-linux/pull/229#discussion_r716890771

**Review by sagie-shamay — APPROVED** [review]

**Pull request merged**